docs/DeveloperGuide.md

```diff
@@ -502,7 +511,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-* 1b. The given profile to add is invalid
+* 1b. The given profile to add is invalid.
 
     * 1b1. NUScheduler shows an error message.
 
```

Comment:
for line 529 below should we change to 1000 by 700px? alternatively can change min size of MainWindow in fxml as well.

Reply:
I agree. Let's update Line 529 instead, changing the window size is probably an enhancement.